### ISSUE TITLE
[Bug]: show create table  mo_columns panic

### ISSUE THREAD

**Issue by dengn** [matrixorigin/matrixone#7322]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): affb0c8332580c96789d648a4ed0c668a05262c1
- Hardware parameters:
- OS type: 
- Others:
```

### Actual Behavior

system table problem.

mysql> use system;
mysql> show create table mo_columns;
ERROR 20101 (HY000): internal error: panic runtime error: invalid memory address or nil pointer dereference: 
runtime.panicmem
        /usr/local/go/src/runtime/panic.go:260
runtime.sigpanic
        /usr/local/go/src/runtime/signal_unix.go:835
github.com/matrixorigin/matrixone/pkg/sql/plan.buildShowCreateTable
        /workspace/mo20221226/matrixone/pkg/sql/plan/build_show.go:99
github.com/matrixorigin/matrixone/pkg/sql/plan.BuildPlan
        /workspace/mo20221226/matrixone/pkg/sql/plan/build.go:98
github.com/matrixorigin/matrixone/pkg/sql/plan.(*BaseOpt

**Comment by aressu1985** [comment]
do not fixed:087b5eb534adb4a22a12f5f957435f0ed4e27386

mysql> use system;
Reading table information for completion of table and column names
You can turn off this feature to get a quicker startup with -A

Database changed
mysql> show create table mo_columns;
ERROR 20101 (HY000): internal error: panic runtime error: invalid memory address or nil pointer dereference: 
runtime.panicmem
	/usr/local/go/src/runtime/panic.go:260
runtime.sigpanic
	/usr/local/go/src/runtime/signal_unix.go:835
github.com/matrixorigin/matrixone/pkg/sql/plan.buildShowCreateTable
	/Users/sudong/Program/MatrixOrigin/matrixone/pkg/sql/plan/build_show.go:105
github.com/matrixorigin/matrixone/pkg/sql/plan.BuildPlan
	/Users/sudong/Program/MatrixOrigin/matrixone/pkg/sql/plan/build.go:101
github.com/matrixorigin/matr

**Comment by taofengliu** [comment]
I can not reproduce it.  It seems OK.
![2023-01-19 16-12-32 的屏幕截图](https://user-images.githubusercontent.com/81315978/213389240-15a9d668-2017-450e-876f-4bf2d7b072ce.png)

**Comment by daviszhen** [comment]
taofeng is on leave about one montg.

**Comment by iamlinjunhong** [comment]
affb0c8332580c96789d648a4ed0c668a05262c1 can reproduce, ea7c3fa906e3b6d15dca44cc387ad1113843b1f6 can't reproduce. It has been solved.

**Comment by aressu1985** [comment]
fixed:46a417daf4b55b93a35682995033ed18b8f1d096
